verilog/tools/kythe/kythe_facts_extractor.cc

```diff
@@ -222,6 +272,28 @@ VName KytheFactsExtractor::ExtractVariableReferenceFact(
   }
 }
 
+VName KytheFactsExtractor::ExtractPackageDeclaration(
+    const IndexingFactNode& package_declaration_node) {
+  const auto& anchors = package_declaration_node.Value().Anchors();
+  const Anchor& package_name = anchors[0];
+  const Anchor& package_end_label = anchors[1];
+
+  const VName package_vname(file_path_,
+                            CreateScopeRelativeSignature(
+                                CreatePackageSignature(package_name.Value())));
+  const VName package_name_anchor = PrintAnchorVName(package_name);
+
+  GenerateFactString(package_vname, kFactNodeKind, kNodePackage);
+  GenerateEdgeString(package_name_anchor, kEdgeDefinesBinding, package_vname);
+
+  if (anchors.size() > 1) {
```

Comment:
You've already referenced `anchors[1]` above.  Should that reference declaration `package_end_label` be moved inside here to protect from out-of-bounds access?